girderformindlogger/web_client/src/templates/body/userAccount.pug

```diff
@@ -22,10 +22,18 @@ ul.g-account-tabs.nav.nav-tabs
 
 .tab-content
   #g-account-tab-info.tab-pane.active
+    .g-user-login-container.g-user-settings-container
+      form#g-user-login-form(role="form")
+        .form-group
+          label(for="g-login") Username
+          input#g-login.input-sm.form-control(type="text", value=user.get('email'))
+        button.btn.btn-default.btn-sm(type="submit")
+          i.icon-edit
+          | Save
+        #g-user-login-error-msg.g-validation-failed-message
```

Comment:
This is good but unnecessary. This interface is vestigial from the upstream Girder project and not one in use for MindLogger.